tests/s-thread-pthread_exit.c

```diff
@@ -0,0 +1,27 @@
+#include <pthread.h>
+
+#define NUM_THREAD 4
+
+pthread_t threads[NUM_THREAD];
+void *thread_main(void *arg)
+{
+	double result;
+	result = 1.0;
+	printf("%f\n",result);
+	pthread_exit((void *) 0);
+}
+
+int main(void)
+{
+	for (int i = 0; i < NUM_THREAD; i++){	
+		pthread_create(&threads[i], NULL, &thread_main, (void *)i);
```

Comment:
The `(void *)i` will complain about the cast.  As it's not used in the thread_main(), you can just pass NULL here.  Also please put a space between ")" and "{".